### DISCUSSION TITLE
任何版本都会重现的一个问题

### DISCUSSION THREAD

**Discussion by supspp** [midoks/mdserver-web#668] [Q&A]
![QQ20241224-190820](https://github.com/user-attachments/assets/e3e694fe-deca-45c3-9a96-5ac45f2057b7)

OpenResty 1.27.1 和任意版本

性能调整参数修改后

点击重载配置，那么会还原已经修改过的参数

php不存在，只有在OpenResty

**Comment by midoks** [comment]
重载:重置为默认配置+并重启。
